src/profile-logic/profile-data.js

```diff
@@ -1507,6 +1507,47 @@ export function isNetworkMarker(marker: TracingMarker): boolean {
   return !!(marker.data && marker.data.type === 'Network');
 }
 
+export function getNetworkMarkers(thread: Thread): Array<NetworkPayload> {
+  const { markers } = thread;
+  const networkMarker = [];
+
+  // making sure we return all markers after we got all from type network
```

Comment:
I'm confused by this comment, and I believe markers will always be non-null, so that code branch will never be hit. Setting up in-editor hints for Flow types is really helpful for stuff like this.

Reply:
Removed it.